swagger.yaml

```diff
@@ -5605,6 +5605,11 @@ definitions:
       rhcos_version:
         type: string
         description: Build ID of the RHCOS image.
+      images:
+        type: object
+        additionalProperties:
+          $ref: '#/definitions/openshift-images'
+        description: RHCOS/Release images and versions for each CPU architectures.
```

Comment:
Am I wrong or it duplicates the above fields?
In case we do need to maintain backwards compatibility, maybe change the name of the field from ``images`` to ``extra_versions`` or maybe ``per_arch_versions``

Reply:
Right, that's should be a map of CPU arch to images, so it includes the same properties for each arch. The current properties are kept for backwards compatibility indeed of old versions. So not sure about the naming, as we'll probably just remove these properties when deleting the old versions. I mean, 'images' should be available only for new versions (> 4.9) instead of the current properties. Maybe 'images_per_arch'? Or just add more explanation in the description it make it clearer.

Reply:
add move explanation to the description and also change the name to the one you suggested
sounds good